x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/select_direction.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+
+import { NetworkTopNFlowDirection } from '../../../../graphql/types';
+
+import { NetworkTopNFlowTableId } from '.';
+import { SelectDirection } from './select_direction';
+
+describe('NetworkTopNFlow Select direction', () => {
+  const mockOnChange = jest.fn();
+
+  describe('rendering', () => {
+    test('it renders the basic group button for uni-direction and bi-direction', () => {
+      const wrapper = shallow(
+        <SelectDirection
+          id={`${NetworkTopNFlowTableId}-select-direction`}
+          selectedDirection={NetworkTopNFlowDirection.uniDirectional}
```

Comment:
Awesome!! Thanks for adding this and the below tests for the uni/bi-directional button! :D Nice improvements from those bugs found as part of the merge earlier this week.